From 17f3df8fd718fb229ae3453ae59b3f2349464d06 Mon Sep 17 00:00:00 2001
From: kernel test robot <lkp@intel.com>
Date: Mon, 16 Aug 2021 22:13:14 -0700
Subject: [PATCH] scsi: qla2xxx: edif: Fix returnvar.cocci warnings
Git-commit: 17f3df8fd718fb229ae3453ae59b3f2349464d06
Patch-mainline: v5.15-rc1
References: git-fixes
Modified-by-SEL: No


drivers/scsi/qla2xxx/qla_edif.c:213:25-29: Unneeded variable: "rval". Return "0"
on line 264

Remove unneeded variable used to store return value.

Generated by: scripts/coccinelle/misc/returnvar.cocci

Link: https://lore.kernel.org/r/20210817051315.2477-12-njavali@marvell.com
Fixes: 7ebb336e45ef ("scsi: qla2xxx: edif: Add start + stop bsgs")
Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: kernel test robot <lkp@intel.com>
Signed-off-by: Nilesh Javali <njavali@marvell.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Jun Yang <jun.yang@suse.com>
---
 drivers/scsi/qla2xxx/qla_edif.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/scsi/qla2xxx/qla_edif.c b/drivers/scsi/qla2xxx/qla_edif.c
index dc10874a0c99..ad746c62f0d4 100644
--- a/drivers/scsi/qla2xxx/qla_edif.c
+++ b/drivers/scsi/qla2xxx/qla_edif.c
@@ -628,7 +628,6 @@ qla_edif_app_start(scsi_qla_host_t *vha, struct bsg_job *bsg_job)
 static int
 qla_edif_app_stop(scsi_qla_host_t *vha, struct bsg_job *bsg_job)
 {
-	int32_t                 rval = 0;
 	struct app_stop         appstop;
 	struct fc_bsg_reply     *bsg_reply = bsg_job->reply;
 	struct fc_port  *fcport, *tf;
@@ -679,7 +678,7 @@ qla_edif_app_stop(scsi_qla_host_t *vha, struct bsg_job *bsg_job)
 
 	/* no return interface to app - it assumes we cleaned up ok */
 
-	return rval;
+	return 0;
 }
 
 static int
-- 
2.34.1

